cad0p/uu-afp-2021-team-proj-abbr
# chore(frontend-elm-starter): Basic Elm demo

# What

Creating a basic ELM demo app.  
Not an extension yet, but just to test that Elm works.  

## How to test it

Can be easily tested with `elm reactor`: 

```bash
$ elm reactor
Go to http://localhost:8000 to see your project dashboard.
```  

Then open the browser, and locate yourself to `src/Main.elm`.  

<img width="410" alt="image" src="https://user-images.githubusercontent.com/12134172/160935992-8ed06650-c4b7-4486-bdf5-43501266a8bb.png">

You should see something like: <https://elm-lang.org/examples/cat-gifs>.  
Keep in mind the free Giphy limit: 42 requests per hour... 

## Next steps

- Get familiar with the ecosystem
- Start writing the extension code
- Setup the GitHub actions with lining and testing (maybe) for FE
